istioctl/pkg/tag/generate.go

```diff
@@ -269,6 +270,10 @@ istiodRemote:
 		decodedWh.Name = webhookName
 	}
 
+	for k, v := range config.Labels {
+		decodedWh.Labels[k] = v
```

Comment:
Just curious what prompted adding this? Seems like a good idea just wasn't sure the motvation. Wondering if annotations are useful as well

Reply:
can `decodedWh.Labels` ever be nil?

Reply:
> can `decodedWh.Labels` ever be nil?

It should not be nil since it has the tag label already, but I think it's better to add the nil check to prevent potential panic.

> Just curious what prompted adding this? Seems like a good idea just wasn't sure the motvation. Wondering if annotations are useful as well

I found that the uninstallation process for the operator is slightly different from istioctl, which requires specific labels. Then, I realized that the labels of the deactivated webhook are not consistent with the generated one. I think it's also a good idea to add the annotation, but I'm not sure if it is useful for the webhook mechanism though.